React/src/screens/Dashboard/screens/Home/index.tsx

```diff
@@ -1,70 +1,47 @@
-import { useTranslation } from 'react-i18next';
 import { useForm } from 'react-hook-form';
 
-import { actionCreators as authActions } from 'contexts/UserContext/reducer';
-import { useDispatch as useUserDispatch } from 'contexts/UserContext';
-import { logout, removeCurrentUserToken } from 'services/AuthService';
-import FormInput from 'components/FormInput';
+import { UserData } from 'interfaces/userData';
 
-import logo from './assets/logo.svg';
-import styles from './styles.module.scss';
-import { withContextProvider, useSelector, useDispatch } from './context';
-import { actionCreators } from './context/reducer';
+import LogoWolox from '../../../../resources/img/logoWolox.png';
```

Comment:
Have you tried using?
```suggestion
import LogoWolox from 'resources/img/logoWolox.png';
```